eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/retry/Retryer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.runtime.core.retry;
+
+import org.apache.eventmesh.runtime.core.timer.TimerTask;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Retryer interface.
+ */
+public interface Retryer {
+
+    void newTimeout(TimerTask timerTask, long delay, TimeUnit timeUnit);
```

Comment:
I think that a better design should be based on implementing an interface (specification) rather than an abstract class, so an interface is added.

---

我觉得, 好的设计顶层应该基于实现某个接口(规范)而不是抽象类来设计, 因此添加了一个接口。